packages/internal/src/createBundles.js

```diff
@@ -57,11 +57,10 @@ export const extractProposalBundles = async (
           const srcPath = req.resolve(src, { paths: [home] });
           if (toBundle.has(bundlePath)) {
             const oldSrc = toBundle.get(bundlePath);
-            assert.equal(
-              oldSrc,
-              srcPath,
-              `${bundlePath} already installed as ${oldSrc}, also given ${srcPath}`,
-            );
+            oldSrc === srcPath ||
+              Fail`${q(bundlePath)} already installed as ${q(
+                oldSrc,
+              )}, also given ${q(srcPath)}`;
```

Comment:
Chip certainly has a point that inserting line breaks around `oldSrc,` makes it harder to read.

Reply:
Yup.  I've gotten pretty liberal with `// prettier-ignore` when it makes a substantial improvement in legibility.

Reply:
I ran into a problem with a `// prettier-ignore` under CI, where it still complained about something not being in canonical format. I just gave up rather than investigating, but I know how to reproduce if you're interested.

Reply:
does anything in createBundles module need quote/censoring? this is run locally before the chain starts.

Reply:
> does anything in createBundles module need quote/censoring? this is run locally before the chain starts.

If nothing in createBundles needs censoring, then everything in createBundles should be quoted.

Reply:
I just looked, and there are no remaining redactions in createBundles.js. Resolving.